lib/ansible/modules/cloud/amazon/aws_s3_bucket_info.py

```diff
@@ -32,7 +34,12 @@
 # Note: Only AWS S3 is currently supported
 
 # Lists all s3 buckets
-- aws_s3_bucket_facts:
+- aws_s3_bucket_info:
+  register: result
+
+- name: List buckets
+  debug:
+    var: "{{ result['buckets'] }}"
```

Comment:
```suggestion
    msg: "{{ result['buckets'] }}"
```

Reply:
Looks like I was somewhat sleepy when trying to "fix" that... Thanks for checking again! This time I really got it ;)